packages/component-submission/server/resolvers/submitManuscript.js

```diff
@@ -31,7 +31,12 @@ async function submitManuscript(_, { data }, { user, ip }) {
 
   manuscript.applyInput(manuscriptInput)
 
-  const sourceFile = manuscript.getSource()
+  logger.info(`Submitting ${manuscript.id}`)
```

Comment:
This logging can probably be removed as part of #1931 ... need to think about what we want in terms of logging.